requirements.txt

```diff
@@ -0,0 +1,9 @@
+cfgv==3.3.1
```

Comment:
Conan less then 2 maybe as well?

Reply:
Yeah this is something I wanted to discuss in the next Shepherds :)

Reply:
For now, let's not have Conan present at first, and if we see the need to have it, we can add it at a later date